src/propsheet/console.h

```diff
@@ -104,7 +105,7 @@ GetAltFaceName(
     LPCTSTR ptszFace
     );
 
-NTSTATUS DestroyDbcsMisc();
+NTSTATUS DestroyDbcsMisc(VOID);

```

Comment:
can this have the nodiscard attribute as well?

Reply:
yeah, oops